spec/requests/audits_requests_spec.rb

```diff
@@ -109,6 +117,13 @@
           post audits_path(default_params.merge(audit: invalid_attributes))
           expect(response).to render_template(:new)
         end
+
+        it "re-renders the 'new' template with an error message when an invalid storage location is given" do
+          post audits_path(default_params.merge(audit: invalid_storage_location_attributes))
+          expect(response).to render_template(:new)
+          expect(flash[:error]).to_not eq("Storage location can't be blank")
```

Comment:
This line isn't necessary - if something is equal to X (next line) by definition it isn't equal to anything but X :grin: